internal/serve/httphandler/profile_handler.go

```diff
@@ -354,16 +354,17 @@ func (h ProfileHandler) GetOrganizationInfo(rw http.ResponseWriter, req *http.Re
 		return
 	}
 
-	distributionPublicKey, err := h.DistributionAccountResolver.DistributionAccountFromContext(ctx)
+	distributionAccount, err := h.DistributionAccountResolver.DistributionAccountFromContext(ctx)
 	if err != nil {
-		httperror.InternalError(ctx, "Cannot get distribution account public key", err, nil).Render(rw)
+		httperror.InternalError(ctx, "Cannot get distribution account", err, nil).Render(rw)
 		return
 	}
 
 	resp := map[string]interface{}{
 		"name":                             org.Name,
 		"logo_url":                         lu.String(),
-		"distribution_account_public_key":  distributionPublicKey,
+		"distribution_account":             distributionAccount,
+		"distribution_account_public_key":  distributionAccount.ID,
```

Comment:
Maybe we can add a TODO and a ticket to remove the public_key ?

Reply:
Good point! Added it in https://github.com/stellar/stellar-disbursement-platform-backend/pull/280/commits/1b9cf230aed431ddb82232e64e491a8937b203f7.